examples/hello-world.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
should these examples also show the expected output of running the tool on the input Ingress?

Reply:
Hey @sunjayBhatia. I think those are just examples we provided to have quickstart of i2gw.

Please see https://github.com/kubernetes-sigs/ingress2gateway/pull/18/files